doc/lib/sphinxutil.py

```diff
@@ -73,6 +73,11 @@ def create_bullet_list(self, lines: [str]):
 
         return bullet_list, parse_msgs
 
+    def create_table_builder(self, table_classes:[str] = ['colwidths-auto']):
```

Comment:
nit: according to python's typing module, the typehin should rather read: `typing.List[str]` (or maybe even better something more generic, such as typing.Iterable[str])